CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+0.23.0 / TBC
```

Comment:
Seems like the date of release is know. If that is the case, can this be updated accordingly?